test/effects/rotate_effect_test.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flame/components/position_component.dart';
+import 'package:flame/effects/effects.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'effect_test_utils.dart';
+
+void main() {
+  const double angleArgument = 6.0;
+  TestComponent component() => TestComponent(angle: 0.5);
+
+  RotateEffect effect(bool isInfinite, bool isAlternating) {
```

Comment:
nit: I think making these named parameters will make the callers more readable

Reply:
Good idea, done!